edweiss-app/components/memento/CardScreenComponent.tsx

```diff
@@ -49,179 +44,109 @@ import { OptionCardModalDisplay } from './OptionCardModalAction';
-                    </RichText>
-                    <TText style={{ position: 'absolute', top: '2%', right: '2%' }} >
-                        <Icon name={mementoStatusIconMap[card.learning_status]} color={mementoStatusColorMap[card.learning_status]} size={25} />
-                    </TText>
-                </Animated.View>
-            </TapGestureHandler>
-
-            <TapGestureHandler
-                testID='flipCardToSeeQuestion'
-                onHandlerStateChange={({ nativeEvent }) => {
-                    if (nativeEvent.state === State.END) {
-                        toggleFlip();
-                    }
-                }}>
-                <Animated.View testID='cardAnswerFace' style={[styles.cardContainer, backCardStyle]}>
-                    {/*<TText mr={10} ml={10} size={20} ellipsizeMode='tail' style={{ textAlign: 'center', fontSize: calculateFontSize(card?.answer ?? ""), lineHeight: calculateFontSize(card?.answer ?? "") * 1.2 }}>
-                        {card.answer}
-                    </TText>*/}
-                    {<RichText color='text' px={'sm'} >
-                        {card.answer}
-                    </RichText>}
-                    <TText style={{ position: 'absolute', top: '2%', right: '2%' }} >
-                        <Icon name={mementoStatusIconMap[card.learning_status]} color={mementoStatusColorMap[card.learning_status]} size={25} />
-                    </TText>
-                </Animated.View>
-            </TapGestureHandler>
+            />
+            <TView style={{ marginTop: '15%' }}>
+                <CardDisplayComponent card={card} />
+            </TView>
 
+            <TView style={styles.container}>
+                {/* Buttons container */}
+                <TView style={styles.buttonContainer}>
+                    <TView style={styles.buttonHalf}>
+                        <FancyButton
+                            mx={0}
+                            testID='notYetButton'
+                            backgroundColor='transparent'
+                            textColor='red'
+                            onPress={async () => {
+                                await updateCard("Not yet");
+                                handleNext();
+                            }}
+                            style={{ width: '100%', height: '100%', borderColor: '#d20f39', borderWidth: 0 }} // Make button fill its half
+                            icon='close-sharp'
+                        >
+                            Not yet
+                        </FancyButton>
+                    </TView>
+                    <TView style={styles.lastButtonHalf}>
+                        <FancyButton
+                            mx={0}
+                            testID='gotItButton'
+                            backgroundColor='transparent'
+                            textColor='green'
+                            onPress={async () => {
+                                await updateCard("Got it");
```

Comment:
Is there no need to handle loading if there is an await function in the button press?

Reply:
Nah since the loading is instantaneous.